news/2018-05-01-twc-semis-recap.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "osu!taiko World Cup - Semifinals Recap"
+date: 2018-05-02 00:00:00 +0000
+---
+Did you miss the past weekend of matches in the osu!taiko World Cup? Don't worry, as Eiuh helps you catch up on what went down in the Semifinals!
+![](/wiki/shared/news/banners/TWC_2018_banner.jpg)
+
+The Semifinals offered some very exciting matches, including the first tiebreaker we've seen since Group Stage. Let's walk through them, shall we?
+
+
+### Semifinals Matches
+
+**The opening match started with Australia eliminating Chile 6-2.** At first, the Australians found themselves in a 1-2 deficit, losing out heavily on the nomods [UNDEAD CORPORATION - Revived](https://osu.ppy.sh/beatmapsets/373517#taiko/818305) and [Qoiet - Out Now On Crowsnest](https://osu.ppy.sh/beatmapsets/667033#taiko/1411524). All the following picks went in favour of Australia, as the combined strength of [Jaye](https://osu.ppy.sh/users/4841352) and [rango](https://osu.ppy.sh/users/7728466) was too much to handle for the Chileans.
+
+The next team to secure their top 8 placement was **Indonesia with a 6-0 sweep over Brazil.** While some maps were close, Brazil leaves the match empty-handed as they're knocked out of the tournament along with Chile, leaving no Latin American teams left in the cup.
+
+**The European brawl between Spain and Poland ended 6-2 for Spain** [CreepyDuck](https://osu.ppy.sh/users/2435013) besting both of the Spaniards on 5 out of 8 maps was simply not enough for Poland. Spain would be the last European team standing, but not for long. 
```

Comment:
missing a dot after Spain

also this sounds like not giving Spain any credit whatsoever lol, also no comment on Canada having two failing scores on Strike in both matches?